### PR TITLE
chore: Add error reporting of caught exceptions

### PR DESCRIPTION
**Pull request by iamareebjamal**
For #4174

I am not even half complete and I give up. It is too much mechanical work

### CODE REVIEW

**Comment by iamareebjamal** [comment]
And there is so so so much code duplication. I have seen 3 sections of deleting event and 2 sections of ordering tickets

File: app/controllers/admin/content/events.js [hunk 1]
```diff
@@ -68,7 +70,8 @@ export default class extends Controller {
             id: 'mode_add_succ'
           });
       })
-      .catch(() => {
+      .catch(e => {
+        console.error('Error while adding ' + camelCasedValue, e);
```
Comment (niranjan94) [inline app/controllers/admin/content/events.js]: Codacy Issue found: [Unexpected string concatenation.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5117968)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5117968&bid=16828373)

File: app/controllers/admin/content/events.js [hunk 2]
```diff
@@ -45,7 +46,8 @@ export default class extends Controller {
             id: 'event_prop_del'
           });
       })
-      .catch(() => {
+      .catch(e => {
+        console.error('Error while deleting ' + modelName, e);
```
Comment (niranjan94) [inline app/controllers/admin/content/events.js]: Codacy Issue found: [Unexpected string concatenation.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5117968)

**Comment by iamareebjamal** [comment]
Disabling the lint rule - https://github.com/eslint/eslint/issues/12042#issuecomment-594479967

**Pull request merged**